exa/__init__.py

```diff
@@ -6,18 +6,73 @@
 #########
 """
 import os
-import tempfile
+import sys
 import logging.config
+
 import yaml
+from traitlets import Unicode, default, validate
+from traitlets.config import Configurable, Application
+
```

Comment:
I think we need to import `Container` and some other imports (or at least disable tests for now).

Reply:
Yeah I will add the CI pipeline as an issue and disable it for now.